lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -367,6 +319,79 @@ class @StudentAdmin
           create_task_list_table @$table_task_history_all, data.tasks
         error: std_ajax_err => @$request_response_error_all.text gettext("Error listing task history for this student and problem.")
 
+  rescore_problem_single: (only_if_higher) ->
+    unique_student_identifier = @$field_student_select_grade.val()
+    problem_to_reset = @$field_problem_select_single.val()
+    if not unique_student_identifier
+      return @$request_response_error_grade.text gettext("Please enter a student email address or username.")
+    if not problem_to_reset
+      return @$request_response_error_grade.text gettext("Please enter a problem location.")
+    send_data =
+      unique_student_identifier: unique_student_identifier
+      problem_to_reset: problem_to_reset
+      only_if_higher: only_if_higher
+    success_message = gettext("Started rescore problem task for problem '<%= problem_id %>' and student '<%= student_id %>'. Click the 'Show Background Task History for Student' button to see the status of the task.")
```

Comment:
Not sure if these messages show up. The label for the task history button has changed.